pyproject.toml

```diff
@@ -111,7 +112,7 @@ ignore-init-module-imports = true
 "coreax/__init__.py" = ["F401"]
 "documentation/source/conf.py" = ["E402"]
 "documentation/source/snippets/*" = ["D100", "F821"]
-"examples/*" = ["PLR0914", "PLR0915"]
+"examples/*" = ["PLR0914", "PLR0915", "PLR6104"]
```

Comment:
Add a comment for why PLR6104 needs to be ignored.

It sounds like the false positive may be due to an instability in new features in Pylint. Add a ticket to come back to this exclusion once Ruff matures.

Reply:
I have added issue #560 to address this. Indeed the false positive appears to result from the new rule not differentiating between `x = x / 2` which can be given by `x /= 2`, and `x = 2 / x`, which cannot be given by `x /= 2`. However, in the specific case here, I have removed the ignore and instead updated the example code to avoid usage of the "magic" divide is join, in favour of `.joinpath` (this avoids the linting issue and enhances the clarity of the examples)